x/ibc/core/04-channel/types/packet.go

```diff
@@ -26,6 +26,9 @@ func CommitPacket(cdc codec.BinaryMarshaler, packet exported.PacketI) []byte {
 	versionHeight := sdk.Uint64ToBigEndian(timeoutHeight.GetVersionHeight())
 	buf = append(buf, versionHeight...)
 
+	delayPeriod := sdk.Uint64ToBigEndian(packet.GetDelayPeriod())
```

Comment:
the comment needs to be updated

Reply:
also nit, I'd prefer to define all the components first and afterward set the stack of appends for better readability